static/js/app.js

```diff
@@ -847,6 +870,95 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            var bool_atrs = {};
+            bool_atrs["show_discussions"] = this.showDiscussion;
```

Comment:
Do you mean associative arrays? I didn't understand what do you think I should change.